css/index.css

```diff
@@ -1,3 +1,509 @@
+    flex-direction: row;
+    justify-content: space-between;
+    margin-right: 34px;
+}
+
+.main-content .top-content .text-content:last-child {
+    margin-right: 0;
+}
+
+.main-content .middle-img {
+    margin-top: 45px;
+}
+
+.main-content .bottom-content {
+    margin-top: 37px;
+    padding: 0 10px;
+}
+
+.main-content .bottom-content .text-content2 {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-right: 12px;
+}
+
+.main-content .bottom-content .text-content2:last-child {
+    margin-right: 0;
+}
+
+.contact {
+    margin-top: 57px;
+    border-top: 1px solid black;
+    padding: 39px 10px 0;
+}
+
+.contact p {
+    margin: 20px 0;
+}
+
+footer {
+    margin: 32px 0;
+    text-align: center;
+    font-size: 14px;
+}
+
+/* Services Page */
+
+.serv img {
+    margin: 20px 0px 20px 0px;
+}
+
+.serv .words {
+    padding-bottom: 30px;
+    margin-bottom: 30px;
+    border-bottom: 1px solid black;
+}
+
+.six-text-boxes button {
+    align-self: center;
+    background: white;
+    margin-top: 12px;
+    padding: 8px 22px;
+    font-size: 12px;
+    border: 1px solid black;
+    cursor: pointer;
+}
+
+.six-text-boxes button:hover {
+    background: black;
+    color: white;
+}
+
+.topthird {
+    display: flex;
+    flex-direction: row;
+}
+
+.middlethird {
+    display: flex;
+    flex-direction: row;
+}
+
+.bottomthird {
+    display: flex;
+    flex-direction: row;
+    border-bottom: 1px solid black;
+}
+
+.img-list {
+    display: flex;
+    flex-direction: row;
+    margin: 40px 0px 80px 0px;
+}
+
+.topthird .first {
+    justify-content: flex-start;
+    background: lightgrey;
+    border: 2px solid black;
+    margin: 20px;
+    padding: 20px;
+}
+
+.topthird .second {
+    justify-content: flex-end;
+    background: lightgrey;
+    border: 2px solid black;
+    margin: 20px;
+    padding: 20px;
+}
+
+.middlethird .third {
+    justify-content: flex-start;
+    background: lightgrey;
+    border: 2px solid black;
+    margin: 20px;
+    padding: 20px;
+}
+
+.middlethird .fourth {
+    justify-content: flex-end;
+    background: lightgrey;
+    border: 2px solid black;
+    margin: 20px;
+    padding: 20px;
+}
+
+.bottomthird .fifth {
+    justify-content: flex-start;
+    background: lightgrey;
+    border: 2px solid black;
+    margin: 20px 20px 40px 20px;
+    padding: 20px;
+}
+
+.bottomthird .sixth {
+    justify-content: flex-end;
+    background: lightgrey;
+    border: 2px solid black;
+    margin: 20px 20px 40px 20px;
+    padding: 20px;
+}
+
+.img-list .seventh {
+    justify-content: flex-start;
+    margin-right: 20px;
+}
+
+.img-list ul {
+    margin: 20px;
+}
+
+.img-list ul li {
+    list-style: disc;
+    padding: 5px;
+}
+
+.img-list img {
+    justify-content: flex-end;
+    margin-left: 20px;
+    align-self:
+}
+
+/* Tablet */
+
+@media (max-width: 800px) {
+    .container {
+        width: 100%;
+        margin: 0 auto;
+    }
+    header {
+        display: flex;
```

Comment:
No need to repeat `display: flex` for header if it was already used previously in the css file, You only want to add rules for things that will change on this resolution.
It applies for all the elements and rules in the application.